examples/cpp/common/Utils.h

```diff
@@ -41,10 +41,6 @@ class RateCounter {
     times.clear();
   }
 
-  unsigned int count() const {
```

Comment:
The method is never called in our code.

On Sat, Feb 28, 2015 at 7:40 PM, Brad Davis <notifications@github.com>
wrote:

> In examples/cpp/common/Utils.h
> <https://github.com/OculusRiftInAction/OculusRiftInAction/pull/72#discussion_r25560490>
> :
>
> > @@ -41,10 +41,6 @@ class RateCounter {
> >      times.clear();
> >    }
> >
> > -  unsigned int count() const {
>
> I don't get why you would remove this. Knowing the count of measured
> frames might be valuable to a caller.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OculusRiftInAction/OculusRiftInAction/pull/72/files#r25560490>
> .
>